### PR TITLE
feat: landing and login

### PR DESCRIPTION
**Pull request by qin-guan**
Landing page would probably be onboarding screen, hmmm.

### CODE REVIEW

File: components/onboarding/Setup.client.vue [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<script setup lang="ts">
+const step = useLocalStorage('onboarding-step', -1)
+
+const steps = [
+  defineAsyncComponent(() => import('./SetupStep01.vue')),
+]
+</script>
+
+<template>
```
Comment (julwrites) [inline components/onboarding/Setup.client.vue]: Is template a Vue thing? It looks kinda like a html template that will get expanded by some other part of the SPC.

File: components/onboarding/Setup.client.vue [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  <section class="py-16">
+    <div class="h-16">
+      <UButton v-if="step > -1" label="Back" color="gray" @click="step--">
```
Comment (julwrites) [inline components/onboarding/Setup.client.vue]: All these UButton and other Ustuff, are they coming from a UI package or is it coming from Nuxt or Vite?

File: components/onboarding/Setup.client.vue [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        <template #leading>
+          <UIcon name="i-tabler-arrow-left" />
+        </template>
+      </UButton>
+    </div>
+
+    <Transition appear>
```
Comment (julwrites) [inline components/onboarding/Setup.client.vue]: Similarly where is Transition coming from?

**Review by julwrites — APPROVED** [review]
I can't quite tell where the routing happens, but maybe i'm missing it? Or does it get handled somewhere outside of the code?

File: components/onboarding/Setup.client.vue (hunk 1 above)
Comment (qin-guan) [reply components/onboarding/Setup.client.vue]: It’s a Vue SFC thing. 

https://vuejs.org/guide/scaling-up/sfc.html#why-sfc

File: components/onboarding/Setup.client.vue (hunk 3 above)
Comment (qin-guan) [reply components/onboarding/Setup.client.vue]: This one is from Vue core. 

https://vuejs.org/guide/built-ins/transition.html#transition

File: components/onboarding/Setup.client.vue (hunk 2 above)
Comment (qin-guan) [reply components/onboarding/Setup.client.vue]: This one is from Nuxt UI. Components are auto imported. 

https://nuxt.com/docs/migration/auto-imports#auto-imports

**Comment by qin-guan** [comment]
I think the auto imports are still manageable for me because I'm used to it. But we can turn it off as well, lmk? @julwrites

**Comment by julwrites** [comment]
> I think the auto imports are still manageable for me because I'm used to it. But we can turn it off as well, lmk? @julwrites 

Yeah i think that would help me better trace and understand the code.

**Comment by qin-guan** [comment]
Okay I think we can track auto import progress in a separate issue

**Comment by qin-guan** [comment]
#10

**Comment by qin-guan** [comment]
Okay I think the functionality is more or less complete

**Comment by qin-guan** [comment]
Also includes upgrade to Nuxt 3.7, but should not introduce any breaking changes

**Comment by qin-guan** [comment]
Seems like Cloudflare complains about exceeding CPU limits for a while after the 3.7 upgrade deployment but I cant repro it anymore, hmmmmm... just writing it down in case it complains about it in the future

**Pull request merged**